### PR TITLE
Add an rfc issue template

### PR DESCRIPTION
**Pull request by zero-24**
### Problem statement

There is no general rfc issue template.

### Proposed solution

Add an general rfc issue template to the github issue templates

### Open questions

- [ ] Any suggestions about the template?
- [ ] Should we add different questions?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rfc.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Start an Request for Comment
```
Comment (brianteeman) [inline .github/ISSUE_TEMPLATE/rfc.md]:
```suggestion
name: Start a Request for Comment
```

File: .github/ISSUE_TEMPLATE/rfc.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+about: Start an Request for Comment discussion
```
Comment (brianteeman) [inline .github/ISSUE_TEMPLATE/rfc.md]:
```suggestion
about: Start a Request for Comment discussion
```

File: .github/ISSUE_TEMPLATE/rfc.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+---
+
+### Problem statement
```
Comment (brianteeman) [inline .github/ISSUE_TEMPLATE/rfc.md]:
```suggestion
### Problem identified
```

**Comment by zero-24** [comment]
Thanks @brianteeman

**Review by wilsonge — APPROVED** [review]

**Comment by HLeithner** [comment]
Thanks, we can update this template anytime we need to, in the mean time I merge it.

**Pull request merged**